packages/math-input/src/components/input/math-input.tsx

```diff
@@ -973,6 +993,10 @@ class MathInput extends React.Component<Props, State> {
                             ref={(node) => {
                                 this.inputRef = node;
                             }}
+                            onFocus={() => {
```

Comment:
Updated this to ensure that we're making proper use of the focus and blur options, so that we can support adding focus styling when a user tabs to focus a math input. As a result, some minor changes were required to the blur and focus methods.